cardano-crypto-class/src/Cardano/Crypto/VRF/Mock.hs

```diff
@@ -35,11 +36,21 @@ instance VRFAlgorithm MockVRF where
     deriving (Show, Eq, Ord, Generic, NoUnexpectedThunks, ToCBOR, FromCBOR)
 
   maxVRF _ = 2 ^ (8 * byteCount (Proxy :: Proxy MD5)) - 1
-  genKeyVRF = SignKeyMockVRF <$> nonNegIntR
+
+  genKeyVRF seed = SignKeyMockVRF sk
+    where
+      sk = runMonadRandomWithSeed seed nonNegIntR
```

Comment:
Isn't this the really shitty `nonNegIntR` in `Cardano.Crypto.Util`?

Reply:
Improved in https://github.com/input-output-hk/cardano-base/pull/113